internal/indexer/typestring_test.go

```diff
@@ -91,6 +91,24 @@ func TestTypeStringEmptyStruct(t *testing.T) {
 	}
 }
 
+func TestTypeStringNameEqualsAnonymousStruct(t *testing.T) {
+	_, f := findDefinitionByName(t, getTestPackages(t), "TestEqualsStruct")
+
+	signature, extra := typeString(f)
+	if signature != "type TestEqualsStruct = struct" {
+		t.Errorf("unexpected type string. want=%q have=%q", "type TestEqualsStruct struct", signature)
```

Comment:
Should this fail? I'd expect this to hit typestring.go:50

Reply:
The tests passed :thinking:

Reply:
Oh the text is wrong here. I'll fix.

Reply:
Ohh yes cool cool